src/renderers/pixi/node.ts

```diff
@@ -55,7 +56,7 @@ export class NodeRenderer<N extends Node, E extends Edge>{
   private badgeSprites: { fill: PIXI.Sprite, stroke: PIXI.Sprite, icon?: PIXI.Sprite, angle: number }[] = []
   private labelContainer = new PIXI.Container() // TODO - create lazily
   private labelSprite?: PIXI.Text
-  private iconSprite?: PIXI.Text
+  private iconSprite?: PIXI.Text | PIXI.Sprite
```

Comment:
Text is a subclass of Sprite, so I think this can just be `iconSprite?: PIXI.Sprite`